bot/src/keeper.ts

```diff
@@ -149,4 +149,10 @@ const participate = async function (auction: AuctionInitialInfo) {
     currentlyExecutedAuctions.delete(auction.id);
 };
 
+export const participate = async function (auctions: AuctionInitialInfo[]) {
+    for (let i = 0; i < auctions.length; i += 1) {
+        await participateInAuction(auctions[i]);
+    }
```

Comment:
No need to be hardcore, you can use `of`  keyword 🙂 
```suggestion
    for(const auction of auctions) {
        await participateInAuction(auction);
    }
```